src/Components/Contributors/InstututionHelper.ts

```diff
@@ -90,20 +90,25 @@ async function getCountryInformationByCountryCodeWithCache(countryCode: string):
 
 async function attemptGettingInstitutionByName(affiliation: Affiliation): Promise<Affiliation | null> {
   if (affiliation.institutionName) {
-    const guessedByNameFromCristinAffiliations = await searchForInstitutionsByNameAndCountry(
-      affiliation.institutionName,
-      SearchLanguage.En,
-      affiliation.countryCode ?? ''
-    );
-    const foundOnlyOneMatch = guessedByNameFromCristinAffiliations.data.length === 1;
-    if (foundOnlyOneMatch) {
-      const institution = guessedByNameFromCristinAffiliations.data[0];
-      return {
-        cristinInstitutionNr: institution.cristin_institution_id,
-        institutionName: institution.institution_name.en ?? institution.institution_name.nb,
-        countryCode: institution.country,
-        isCristinInstitution: institution.cristin_user_institution,
-      };
+    try {
+      const guessedByNameFromCristinAffiliations = await searchForInstitutionsByNameAndCountry(
+        affiliation.institutionName,
+        SearchLanguage.En,
+        affiliation.countryCode ?? ''
+      );
+      const foundOnlyOneMatch = guessedByNameFromCristinAffiliations.data.length === 1;
+      if (foundOnlyOneMatch) {
+        const institution = guessedByNameFromCristinAffiliations.data[0];
+        return {
+          cristinInstitutionNr: institution.cristin_institution_id,
+          institutionName: institution.institution_name.en ?? institution.institution_name.nb,
+          countryCode: institution.country,
+          isCristinInstitution: institution.cristin_user_institution,
+        };
+      }
+    } catch (_error) {
```

Comment:
hvorfor underscore ?

Reply:
underscore indikerer at man ignorer objektet. Jeg bruker ikke error objektet til noe senere. kan fjerne underscoren hvis det er forvirrende.

Reply:
Synes ikke vi trenger den. Jeg kjenner ikke til den notasjonen. Fra gammelt av et jeg vant til å bruke underscore for å markere temp-objekter. Spesielt i tilfeller som denne `someFunction(_foo) {this.foo = _foo}` Men trodde man stort sett hadde gått bort i fra det

Reply:
Fikset